benchmark/agbenchmark/__main__.py

```diff
@@ -1,33 +1,45 @@
 import glob
 import json
+import logging
 import os
 import sys
 from datetime import datetime, timezone
 from pathlib import Path
-from typing import Any, Optional
+from typing import Any, Optional, Sequence
 
 import click
-import pytest
-import toml
+from click_default_group import DefaultGroup
 from dotenv import load_dotenv
-from helicone.lock import HeliconeLockManager
 
-from agbenchmark.app import app
-from agbenchmark.reports.ReportManager import SingletonReportManager
-from agbenchmark.utils.data_types import AgentBenchmarkConfig
+from agbenchmark.config import AgentBenchmarkConfig
+from agbenchmark.utils.logging import configure_logging
 
 load_dotenv()
 
+try:
+    if os.getenv("HELICONE_API_KEY"):
+        import helicone
+
+        helicone_enabled = True
+    else:
+        helicone_enabled = False
+except ImportError:
+    helicone_enabled = False
```

Comment:
Should this be in `AgentBenchmarkConfig`?

Reply:
It's used 12 lines down in the same file. Still not ideal, but not as bad as it was.